gog/graph.go

```diff
@@ -158,6 +158,15 @@ func (v *astVisitor) Visit(node ast.Node) (w ast.Visitor) {
 			return nil
 		}
 
+	case *ast.SelectorExpr:
+		if sel := v.typesInfo.Selections[n]; sel != nil {
+			t := sel.Recv()
+			if ptr, ok := t.(*types.Pointer); ok {
+				t = ptr.Elem()
```

Comment:
Btw: I have no idea why `*types.Var` only has `IsField()`, but no reference back to the structure the field is defined on. That's why I have to build this map myself.